package.json

```diff
@@ -9,7 +9,7 @@
     "decode",
     "stream"
   ],
-  "version": "0.2.1",
+  "version": "0.2.2",
```

Comment:
Please don't bump the version number, we will do that as part of the release process so that it gets its own commit and tag...